ui/src/app/base/components/TableConfirm/TableConfirm.tsx

```diff
@@ -0,0 +1,88 @@
+import type { ReactNode } from "react";
+
+import {
+  ActionButton,
+  Button,
+  Col,
+  Notification,
+  Row,
+} from "@canonical/react-components";
+import type { Props as ButtonProps } from "@canonical/react-components/dist/components/Button";
+
+import { COL_SIZES } from "app/base/constants";
+import { useCycled } from "app/base/hooks";
+
+export type Props = {
+  confirmAppearance?: ButtonProps["appearance"];
+  confirmLabel: string;
+  errors?: string | Record<string, string[]>;
```

Comment:
Should we be using the `FormErrors` type here? Maybe it needs to go in base types

Reply:
Hmm yeah maybe, or maybe the `FormErrors` type should eventually become a generic type for the API error responses (kind of like I did for the setDefaultErrors case).